example/spec/parallel/discoverGranules/DiscoverGranulesHttpsSpec.js

```diff
@@ -72,7 +72,10 @@ describe('The Discover Granules workflow with https Protocol', () => {
         await waitForGranuleAndDelete(
           config.stackName,
           granule.granuleId,
-          'completed'
+          'completed',
+          {
+            retries: 5,
+          }
```

Comment:
Added this retry because this was failing when granules still had the `queued` status

Reply:
So it wasn't switching the granules from queued to completed fast enough and that was why it failed? 
Because it was trying to check for complete?